draft-02.xml

```diff
@@ -503,22 +503,31 @@
             dereferencable URL that points to human-readable text. If
             the 'format' property appears it SHOULD contain one of the
             following values: 'text', 'html', 'asciidoc', or 'markdown'.
+            Independently of the format specified, the contained text
```

Comment:
uh, the errors of non-native speakers. I haven't done pull requests before, can I edit this one (sort of "edit this pull request") or do I have to submit a completely new pull request?